processor/transformprocessor/internal/common/functions.go

```diff
@@ -33,6 +33,9 @@ func Functions[K any]() map[string]ottl.Factory[K] {
 		ottlfuncs.NewDeleteMatchingKeysFactory[K](),
 		ottlfuncs.NewMergeMapsFactory[K](),
 		ottlfuncs.NewLogFactory[K](),
+		ottlfuncs.NewFnvFactory[K](),
+		ottlfuncs.NewSHA1Factory[K](),
+		ottlfuncs.NewSHA256Factory[K](),
```

Comment:
Please remove this and add it to its own PR or leave a review on https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/23190

Reply:
Added a comment to the other PR https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/23190/files#r1221861948. Will remove this

Reply:
Removed this change